Jenkinsfile

```diff
@@ -68,7 +68,7 @@ pipeline {
             parallel {
                 stage ('Linux') {
                     steps {
-                        withMaven(maven: 'Maven 3.5.4', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                        withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
```

Comment:
❔ are we consistently using `maven-latest` as the label for this version across repositories?

Reply:
I will be using maven-latest where upgrading. It was available but not used. It currently points to 3.6.3, which is the current latest version.